### ISSUE TITLE
What's the training setting for EVA2.0?

### ISSUE THREAD

**Issue by Sylarair** [thu-coai/EVA#51]
Hello, I wonder what the settings are when performing the EVA2.0 pre-training, for example, the batch size, epoch number, the number of used GPU, and training time. Could anyone know the answer to this question? Many thanks!

**Comment by t1101675** [comment]
batch size = 4096
64 NVIDIA V100 32G GPUs
learning rate: 0.01
warmup step: 10K
lr_scheduler: Noam (reverse square root)
max pre-training steps: 400K
training time: ~30 Days

(sorry we didn't record the epoch number because we care more about the total optimization steps)

**Comment by Sylarair** [comment]
Got it. Thanks!
